ambry-admin/src/main/java/com.github.ambry.admin/AdminBlobStorageService.java

```diff
@@ -167,30 +170,12 @@ public void handleGet(RestRequest restRequest, RestResponseChannel restResponseC
 
   @Override
   public void handlePost(RestRequest restRequest, RestResponseChannel restResponseChannel) {
-    long processingStartTime = System.currentTimeMillis();
-    long preProcessingTime = 0;
     handlePrechecks(restRequest, restResponseChannel);
     restRequest.getMetricsTracker().injectMetrics(adminMetrics.postBlobMetrics);
```

Comment:
Actually, I think this is required so that we track rate at the very least - i.e. it should be 0. Also, this is still a request-response cycle and we need to know how long the whole thing is taking.